### PR TITLE
Add opening/closing hours of event

### PR DESCRIPTION
**Pull request by pjotrek-b**
As discussed.
btw: I've also removed trailing whitespace :wink:

### CODE REVIEW

**Review by JeromeMartinez — APPROVED** [review]

File: notimetowait2.md [hunk 1]
```diff
@@ -41,10 +41,13 @@ The event will also present on the current state of the IETF’s [CELLAR](https:
...
 Thursday, November 9, 2017 and Friday, November 10, 2017
 
+Opening: 09:00
+Closing: 17:00 (sharp)
+
 ## Where
 [Österreichisches Filmmuseum - Austrian Film Museum](https://www.filmmuseum.at)
 
-![Österreichisches Filmmuseum - Austrian Film Museum](/MediaConch/images/Filmmuseum_04s.jpeg)  
+![Österreichisches Filmmuseum - Austrian Film Museum](/MediaConch/images/Filmmuseum_04s.jpeg)
```
Comment (dericed) [inline notimetowait2.md:50]: This double space at the end prevents the next line from wrapping into this one.

**Review by dericed — CHANGES_REQUESTED** [review]
Leave the double space after the image

File: notimetowait2.md (hunk 1 above)
Comment (JeromeMartinez) [reply notimetowait2.md:50]: +1

**Review by JeromeMartinez — CHANGES_REQUESTED** [review]
double space

**Comment by JeromeMartinez** [comment]
@pjotrek-b ping ;-) (I just got a question about it :) )

**Comment by dericed** [comment]
please close, resolved by https://github.com/MediaArea/MediaConch-Website/pull/89

**Pull request closed without merging**